app/controllers/alchemy/admin/pages_controller.rb

```diff
@@ -1,3 +1,4 @@
+# coding: utf-8
```

Comment:
Please remove these. They are not necessary in Ruby 2 (Ruby 1.9 support was dropped long ago)